assets/js/base/components/product-grid/style.scss

```
@@ -0,0 +1,236 @@
+@keyframes spin {
+	from {
+		transform: rotate(0deg);
+	}
+	to {
+		transform: rotate(360deg);
+	}
+}
+
+.wc-block-grid__products {
+	display: flex;
+	flex-wrap: wrap;
+	list-style: none;
+	padding: 0;
+	margin: 0 (-$gap/2) $gap;
+	background-clip: padding-box;
+}
+
+.wc-block-grid__product {
+	box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+	float: none;
+	width: auto;
+	position: relative;
+	text-align: center;
+	border-left: $gap/2 solid transparent;
+	border-right: $gap/2 solid transparent;
+	border-bottom: $gap solid transparent;
+}
+
+.wc-block-grid__product-link {
+	text-decoration: none;
+	border: 0;
+	box-shadow: none !important;
+}
+
+.wc-block-grid__product-image {
+	text-decoration: none;
+	margin-bottom: $gap;
+	display: block;
+
+	img {
+		vertical-align: middle;
+		margin-left: auto;
+		margin-right: auto;
+	}
+}
+
+.wc-block-grid__product-title {
+	line-height: 1.2;
+	font-weight: 700;
+}
+
+.wc-block-grid__product-title,
+.wc-block-grid__product-price,
+.wc-block-grid__product-rating {
+	margin-bottom: $gap-small;
+	display: block;
+}
+
+.wc-block-grid__product-price {
+	.wc-block-grid__product-price__regular {
+		font-size: 0.8em;
+		line-height: 1;
+		color: #aaa;
+		margin-top: -0.25em;
+		display: block;
+	}
+	.wc-block-grid__product-price__value {
+		letter-spacing: -1px;
+		white-space: nowrap;
+		font-weight: 600;
+		display: block;
+		font-size: 1.25em;
+		line-height: 1.25;
+		color: #000;
+	}
+}
+
+.wc-block-grid__product-add-to-cart {
+	margin: 0;
+	word-break: break-word;
+	white-space: normal;
+
+	a,
+	button {
+		word-break: break-word;
+		white-space: normal;
+		margin: 0 !important;
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+
+		&.loading {
+			opacity: 0.25;
+		}
+
+		&::after {
+			margin-left: 0.5em;
+			display: inline-block;
+		}
+
+		&.added::after {
+			font-family: WooCommerce; /* stylelint-disable-line */
+			content: "\e017";
+		}
+
+		&.loading::after {
+			font-family: WooCommerce; /* stylelint-disable-line */
+			content: "\e031";
+			animation: spin 2s linear infinite;
+		}
+	}
+}
+
+.wc-block-grid__product-onsale {
+	border: 1px solid #43454b;
+	color: #43454b;
+	background: #fff;
+	padding: 0.202em 0.6180469716em;
+	font-size: 0.875rem;
+	text-align: center;
+	text-transform: uppercase;
+	font-weight: 600;
+	display: inline-block;
+	width: auto;
+	border-radius: 3px;
+	z-index: 9;
+	margin: 0 auto $gap;
+}
+
+.has-image {
+	.wc-block-grid__product-onsale {
+		top: $gap-smaller;
+		right: $gap-smaller;
+		position: absolute;
+	}
+}
+
+.wc-block-grid__product-rating {
+
+	.wc-block-grid__product-rating__stars {
+		overflow: hidden;
+		position: relative;
+		width: 5.3em;
+		height: 1.618em;
+		line-height: 1.618;
+		font-size: 1em;
+		font-family: star; /* stylelint-disable-line */
+		font-weight: 400;
+		display: -block;
```

Should this be `inline-block`?
```suggestion
		display: inline-block;
```